[CS2103T-W11-4] CaloFit

FelixNWJ opened the pull request.

Alaete commented on docs/DeveloperGuide.adoc:
```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
-+
+1.	User enters meal information.
```
What is `user`?

Alaete commented on docs/DeveloperGuide.adoc:
```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
-4.  AddressBook deletes the person
-+
+1.	User enters meal information.
+2.	System adds entry into database.
```
What is `system`?

KartikeKhandelwal commented on docs/DeveloperGuide.adoc:
```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
+|&#11088;&#11088;&#11088;
+|user
+|change my daily calorie budget
+|better reflect my new lifestyle requirements.
```
(Reviewing for tutorial exercise) Can the "So that I can..." part for this can be more specific?

KartikeKhandelwal commented on docs/DeveloperGuide.adoc:
```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
+
+|&#11088;
+|user
+|create a new meal entry based of a meal I commonly eat
```
Shouldnt this be I regularly eat*

Alaete commented on docs/DeveloperGuide.adoc:
```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
+1.	User enters meal information.
+2.	System adds entry into database.
+3.	System shows updated calorie budget for the day.
 Use case ends.
```
Perhaps you should put a `+` before the `Use case ends.` so that it appears on a new line.

Alaete replied:
Before you end your use case, perhaps you should also display information about the previous meal? (This comment is made based on your mockup. :+1:)

Alaete commented on docs/DeveloperGuide.adoc:
```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
+** 1a1.	CaloFit shows an error message.
 +
-Use case resumes at step 2.
+Use case restarts at step 1.
```
Shouldn't the use case ends here? You are going to be stuck in a loop forever if you don't have the necessary information for any meal.

KartikeKhandelwal commented on docs/DeveloperGuide.adoc:
```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
+== Non-functional Requirements
+.	Should work on any mainstream OS as long as it has Java 11 or above installed.
+.	Should be able to hold up to 1000 meal entries without a noticeable sluggishness in performance for typical usage.
+.	A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
```
Maybe this can be more concise, perhaps change to:  Users with higher typing speed can use the interface more efficiently ?

Alaete commented on docs/DeveloperGuide.adoc:
```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
```
Perhaps instead of removing this line, you should edit it to fit your purpose?

Alaete commented on docs/DeveloperGuide.adoc:
```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
-[discrete]
-=== Use case: Delete person
+// [discrete]
+=== Use Case: Add meal entry
```
Do you think that having a use case ID could make it easier to refer to this use case in the future?

KartikeKhandelwal commented on docs/DeveloperGuide.adoc:
```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
+The app is able to help the user keep track of their calorie consumption. This way, they can ensure that they keep to their own diet plans.
 
 [appendix]
 == User Stories
```
Can there be a user story that involves storing and displaying the calorie count for regularly had meals before user decides on what to eat?

KartikeKhandelwal replied:
Additionally, it might be helpful to a user if calories burnt were counted too, so that a daily calorie deficit/surplus can be kept track of.

Alaete commented on docs/DeveloperGuide.adoc:
```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
-+
+1.	User enters meal information.
+2.	System adds entry into database.
+3.	System shows updated calorie budget for the day.
```
What is calorie budget exceeded? You might consider having an extension for this case? 😅

KartikeKhandelwal reviewed (commented):
Most of the necessary user stories are present with the correct format. Generally, well done!

Alaete reviewed (commented):
Good job overall 💯. Here are some suggestions that you can consider for your use case 😸.

krusagiz commented on docs/DevGuideFeatureImplementations/budgetbar.adoc:
```diff
@@ -0,0 +1,53 @@
+** Each meal is displayed as a proportionally-sized segment in the whole bar.
+* Position of warning/danger markers
+
+The following activity diagram shows the reactive update dependencies.
```
Is it an activity diagram?

krusagiz commented on docs/DeveloperGuide.adoc:
```diff
@@ -142,105 +143,223 @@ image::StorageClassDiagram.png[]
+*** The dish will then be added to both the dish database as well as the meal log
 
-image::UndoRedoState1.png[]
+The following sequence diagram shows how the `add` operation works when calorie information is included:
```
Perhaps you could label the `[else]` condition in the `alt` block?

krusagiz commented on docs/DevGuideFeatureImplementations/AddFeatureGuide.adoc:
```diff
@@ -0,0 +1,44 @@
+*** The dish will be added to the meal log only.
+
+** The user inserts the calorie information different from the calorie in the dish database
+*** This will create a new dish all together as there are differences in the calorie
```
Should it be **altogether** instead?

eugenood commented:
Here's a few comments from our team on your DG:

1. Did you forget to change your `LogicClassDiagram.png`?
2. What do you mean when you say that storing a list of `Tag` in `ModelManager` is more OO?
3. Do you perhaps want to use markup for your `UNKNOWN_CALORIE` in section 3.1.1?
4. Do you mean class or object diagram in section 3.2.1?
5. Do you want to fix up your class or object diagram in section 3.2.1? The information included is a little sparse.
6. Do you perhaps want to include the little rectangle thingy for the self-calls in your sequence diagram in 3.3.1?

krusagiz commented on docs/DevGuideFeatureImplementations/AddFeatureGuide.adoc:
```diff
@@ -0,0 +1,44 @@
+
+There are a few cases to take note:
+
+* If the dish is in the dish database:
```
Would you like to use an Activity diagram to illustrate the alternative paths instead of a Sequence diagram? It seems like most of your Sequence diagram is not directly related to your description?

krusagiz commented on docs/DevGuideFeatureImplementations/AddFeatureGuide.adoc:
```diff
@@ -0,0 +1,44 @@
+
+* If the dish is in the dish database:
+** The user inserts the calorie information that is the same as the one in the dish database
+*** The dish will be added to the meal log only.
```
Do you want to consider standardising your punctuation (i.e. your full stops and colons)?

krusagiz commented on docs/DevGuideFeatureImplementations/suggest.adoc:
```diff
@@ -0,0 +1,50 @@
+
+Step 3. The success message will be return, while printing the updated dish list on CaloFit.
+
+The following sequence diagram shows how the suggest operation works:
```
Is `deduct c from remainingBudget` a method call, should it be in a sequence diagram? 

Are the self invocation methods drawn correctly? It doesn't seem clear?

FelixNWJ committed:
Merge pull request #84 from MauriceTXS/notification

Update auto Suggest feature

MauriceTXS committed:
Merge pull request #81 from FelixNWJ/dev-guide

Developer guide update for Report feature

fangpinsern committed:
Add Command Change

Allow add from suggest/find panel

MauriceTXS committed:
Updated Suggest feature

fangpinsern committed:
Merge pull request #85 from fangpinsern/master

Add Command Change

MauriceTXS committed:
Added EOL

MauriceTXS committed:
Merge branch 'master' into notification

MauriceTXS committed:
Updated new picture

MauriceTXS committed:
Updated UserGuide

FelixNWJ committed:
Merge pull request #87 from MauriceTXS/notification

Updated Suggest feature and UserGuide

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

FelixNWJ committed:
Merge branch 'master' into report-v1.3

FelixNWJ committed:
Merge branch 'master' into dev-guide

FelixNWJ committed:
Fix bug in closing ReportWindow

FelixNWJ committed:
Update Statistics test

fangpinsern committed:
Fixed Bugs in Add Command

User can now add using just name and tag with the tags reflected in the meal log

FelixNWJ committed:
Merge pull request #89 from fangpinsern/master

Fixed Bugs in Add Command

FelixNWJ committed:
Update PPP questions

FelixNWJ committed:
Update Statistics test and add ReportWindow barchart for Calories over time

FelixNWJ committed:
Add JsonAdaptedMeal and JsonMealLogStorageTest

FelixNWJ committed:
Add JsonSerializableMealLog Test class and add Timestamp equality check

FelixNWJ committed:
Fix checkstyle

FelixNWJ committed:
Update JavaDocs

FelixNWJ committed:
Update Developer Guide Report Feature section

fangpinsern committed:
Merge pull request #91 from FelixNWJ/report-v1.3

ReportWindow update and Test class addition

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

FelixNWJ committed:
Merge branch 'master' into report-v1.3

FelixNWJ committed:
Merge branch 'master' into dev-guide

FelixNWJ committed:
Fix Math bug in calculating statistics average value

FelixNWJ committed:
Refactor Statistics generation for abstraction and add logging and assertions for Report feature

MauriceTXS committed:
Merge pull request #92 from FelixNWJ/report-v1.3

Solve bug in statistics and refactored, improved code quality

fangpinsern committed:
Add command result window message updated

fangpinsern committed:
Resolve erroe when add invalid meal index

FelixNWJ committed:
Merge pull request #105 from fangpinsern/master

Add command result window message updated

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

FelixNWJ committed:
Merge branch 'master' into report-v1.3

FelixNWJ committed:
Edit ReportWindow fxml to solve tiny visual bug of misalignment in window

fangpinsern committed:
Allow for 0 calories input and updated user guide

Included suggest in summary as well as corrected the suggest command typo
Allow for input of 0 calories. However, negative numbers are still not allowed.
*Still no check for integer overflow

fangpinsern committed:
Resolve missing fullstop in user guide and pass CheckStyle

Pass CheckStyle
Add fullstop to add command in UserGuide

MauriceTXS committed:
Merge pull request #128 from fangpinsern/master

Allow for 0 calories input and updated user guide

fangpinsern committed:
Add command bug fixes

Adding dishes with the same calories and the same name will give the right output.

MauriceTXS committed:
Merge pull request #130 from fangpinsern/master

Add command bug fixes

FelixNWJ committed:
Solve minor GUI alignment issue and change pie chart for food data to bar chart

MauriceTXS committed:
Changed notification to run every 10 mins and fix suggestion message.

fangpinsern committed:
Merge pull request #131 from MauriceTXS/master

Changed notification to run every 10 mins and fix suggestion message.

MauriceTXS committed:
Updated notifcaiton feature and user guide.

MauriceTXS committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/main

iltep64 committed:
Add a central Timer class handling periodic updates.
This is used for the UI notifications and meal log/budget today values.

MauriceTXS committed:
Merge pull request #133 from iltep64/master

Add a central Timer class handling periodic updates.

MauriceTXS committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/main

# Conflicts:
#	src/main/java/calofit/ui/MainWindow.java

MauriceTXS committed:
Update notification feature.

FelixNWJ committed:
Merge pull request #132 from MauriceTXS/master

Updated Notification feature and user guide.

fangpinsern committed:
Add and delete command updated

Add command can not add multiple items using their indexes
Delete command can also delete multiple items using thier indexes.

Tests incomplete

FelixNWJ committed:
Merge pull request #138 from fangpinsern/master

Add and delete command updated

iltep64 committed:
Add a caching observable mapping, to reduce invalidations on timer tick.

FelixNWJ committed:
Solve statistics bug showing up even when different month

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

FelixNWJ committed:
Merge branch 'master' into report-v1.3

# Conflicts:
#	src/main/java/calofit/model/meal/MealLog.java

FelixNWJ committed:
Fix slight error in static method

fangpinsern committed:
Merge pull request #139 from FelixNWJ/report-v1.3

Solve bugs

fangpinsern committed:
Help window website has been updated

Updated help window website.

iltep64 committed:
Make Timer use ScheduledThreadPoolExecutor instead of Timer.
This avoids issues when user changes the system time.

fangpinsern committed:
Added delete command to developer guide.

Delete command implementation added into developer guide

FelixNWJ committed:
Merge pull request #140 from fangpinsern/master

Help window website has been updated

FelixNWJ committed:
Merge pull request #141 from iltep64/master

Make timer updates more efficient

fangpinsern committed:
Added diagrams to delete

Added sequence diagram and activity diagram

fangpinsern committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/main

MauriceTXS committed:
Update suggest developer and user guide.

FelixNWJ committed:
Merge pull request #142 from MauriceTXS/suggest

Update suggest developer and user guide.

fangpinsern committed:
Editted error messages for add edit and delete command

add edit and delete commands error messages have been editted

fangpinsern committed:
Improved User guide add command.

Add command in user guide is updated with examples and images

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

FelixNWJ committed:
Merge branch 'master' into report-v1.3

FelixNWJ committed:
Set Name to max limit of 30 characters, add Test for max limit, and update user guide

MauriceTXS committed:
Merge pull request #145 from FelixNWJ/report-v1.3

Set Maximum dish length to 30 as a user constraint

fangpinsern committed:
Checkstyle update

fangpinsern committed:
Add formating edit

fangpinsern committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/main

Resolve merge conflict

fangpinsern committed:
Add, Edit and delete index out of bound error message change

fangpinsern committed:
Undo the command error messages

Will redo it as there is an issue with parsing the add command

fangpinsern committed:
Updated user guide for find command and delete command

User guide for find command and delete command updated with images

FelixNWJ committed:
Merge pull request #143 from fangpinsern/master

Updated developer guide for delete feature

FelixNWJ committed:
Update ReportWindow GUI to prevent overflow of text in the event of many most consumed dishes

fangpinsern committed:
Edited add and delete command error messages

Add no longer allows for adding multiple names into the system.
Both add and delete have new error messages to better handle the index out of bound issue.

fangpinsern committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/main

FelixNWJ committed:
Fix checkstyle error

FelixNWJ committed:
Update Json MealLog and Meal tests

FelixNWJ committed:
Fix test checkstyle

FelixNWJ committed:
Merge pull request #146 from fangpinsern/master

Edited add and delete command error messages

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

FelixNWJ committed:
Merge branch 'master' into report-v1.3

FelixNWJ committed:
Add Test for invalid calorie when creating dish

fangpinsern committed:
Merge pull request #147 from FelixNWJ/report-v1.3

Update ReportWindow GUI and tests for Json Meal, MealLog

FelixNWJ committed:
Checked code standard violations and update java docs

fangpinsern committed:
Add command error message updated

Calories only accept integers

FelixNWJ committed:
Update Report Feature in Developer Guide

FelixNWJ committed:
Slight refactor to make a boolean variable adhere to coding name standard

fangpinsern committed:
Merge pull request #148 from FelixNWJ/report-v1.3

Update JavaDoc and Tests

fangpinsern committed:
Merge pull request #149 from FelixNWJ/dev-guide

Developer Guide Report Feature update and slight refactor in CommandResult class.

FelixNWJ committed:
Update User Guide Report section

FelixNWJ committed:
Add Pictures for Report User Guide

FelixNWJ committed:
Change tone of Report feature to be more user friendly

FelixNWJ committed:
Merge pull request #150 from fangpinsern/master

Add command error message updated

MauriceTXS committed:
Merge pull request #151 from FelixNWJ/dev-guide

Update User Guide Report section

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

FelixNWJ committed:
Merge branch 'master' into dev-guide

FelixNWJ committed:
Move Report User Guide images to correct directory

MauriceTXS committed:
Merge pull request #152 from FelixNWJ/dev-guide

Report Images directory change

iltep64 committed:
Add tokenizeAny and checkAllowedPrefixes to ArgumentMultimap and ArgumentTokenizer.
This is for detection of unknown prefixes.

FelixNWJ committed:
Merge pull request #153 from iltep64/master

Add tokenizeAny and checkAllowedPrefixes

FelixNWJ committed:
Trimmed picture for neatness

FelixNWJ committed:
Upload PPP for FelixNWJ

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3 into PPP

FelixNWJ committed:
Update PPP

MauriceTXS committed:
Merge pull request #154 from FelixNWJ/dev-guide

Trimmed picture for neatness

MauriceTXS committed:
Merge pull request #155 from FelixNWJ/PPP

PPP for FelixNWJ

fangpinsern committed:
Merge

fangpinsern committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/main

fangpinsern committed:
PPP uploaded

fangpinsern committed:
Asicdoctor

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

iltep64 committed:
Terminate timer tasks upon app exit.
This prevents tasks blocking JVM exit when the UI closes.

iltep64 committed:
Add storage for CalorieBudget.

MauriceTXS committed:
Updated notification to show when start up and enlarge notification to be more noticable.

MauriceTXS committed:
Merge branch 'master' into notification

MauriceTXS committed:
Clear unncessary code.

iltep64 committed:
Add hover tooltips for meal bar segments.

iltep64 committed:
Use COMMAND_WORD mechanism for SetBudgetCommand.

iltep64 committed:
Change Calorie parsing mechanism to verify+parse in one method.
This is to avoid having to re-verify the input when parsing.

iltep64 committed:
More tests for SetBudgetCommand/Parser, CommandParser, LogicManager, Tag

iltep64 committed:
Add more tests for CalorieBudget-related and ObservableUtil classes.

MauriceTXS committed:
Updated Find, Suggest, Notificaiton feature developer guide. Group all three features together as assistance.

MauriceTXS committed:
Merge pull request #156 from fangpinsern/master

PPP added

MauriceTXS committed:
Merge branch 'dev-guide' into notification

MauriceTXS committed:
Update minor spelling error.

MauriceTXS committed:
Updated suggest command.

MauriceTXS committed:
Updated notification user guide.

iltep64 committed:
CheckStyle fixes.

fangpinsern committed:
Merge pull request #157 from MauriceTXS/notification

Updated Notification feature

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

FelixNWJ committed:
Added test in StorageManager and changed Report and Statistics tests to reflect actual/ expected parameters correctly.

MauriceTXS committed:
Updated Developer and User guide.

MauriceTXS committed:
Fixed checkstyle

FelixNWJ committed:
Merge pull request #159 from MauriceTXS/master

Fixed bug on notification.

FelixNWJ committed:
Update User Guide Report Feature and Quick Start

MauriceTXS committed:
Fixed bug on notification.

MauriceTXS committed:
Change file location.

MauriceTXS committed:
Corrected spelling error

MauriceTXS committed:
Merge branch 'notification'

MauriceTXS committed:
Updated developer and user guide.

FelixNWJ committed:
Update Developer Guide, User Guide, and PPP

FelixNWJ committed:
Merge pull request #160 from MauriceTXS/master

Updated developer and user guide.

MauriceTXS committed:
Merge pull request #161 from FelixNWJ/master

Update docs

FelixNWJ committed:
Edit tagging in Developer and User Guide

FelixNWJ committed:
Fix grammer and insert Figure labels in Developer Guide

FelixNWJ committed:
Change markup in DeveloperGuide to highlight Classes/Objects and hightlight with italics for Method calls

FelixNWJ committed:
Formatted Developer Guide by italicising Figure descriptions and adding white space where relevant

FelixNWJ committed:
Update PPP to indicate Developer Guide markup changes

MauriceTXS committed:
Merge pull request #162 from FelixNWJ/master

Update docs

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

MauriceTXS committed:
Updated PPP

MauriceTXS committed:
Merge branch 'notification'

MauriceTXS committed:
Updated user guide tag

MauriceTXS committed:
Updated PPP

MauriceTXS committed:
Updated PPP

MauriceTXS committed:
Updated PPP

FelixNWJ committed:
Merge pull request #163 from MauriceTXS/master

Updated PPP and user guide

MauriceTXS committed:
Created test case for suggest, updated bug for notification.

MauriceTXS committed:
Merge branch 'master' into test-case

MauriceTXS committed:
Updated checkstyle for test cases

FelixNWJ committed:
Merge pull request #165 from MauriceTXS/test-case

Test cases for suggest.

MauriceTXS committed:
Merge pull request #164 from FelixNWJ/master

Update Developer Guide Architectural and Introduction sections

MauriceTXS committed:
Updated notification.

FelixNWJ committed:
Merge pull request #166 from MauriceTXS/test-case

Updated notification

iltep64 committed:
Update UG, DG and PPP.

MauriceTXS committed:
Updated developer guide on manual test case.

MauriceTXS committed:
Updated User Guide

FelixNWJ committed:
Merge pull request #167 from MauriceTXS/dev-guide

Update user guide

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

iltep64 committed:
Merge branch 'master' into master

MauriceTXS committed:
Updated Developer guide and PPP

MauriceTXS committed:
Merge branch 'master' into dev-guide

FelixNWJ committed:
Change Developer Guide markup

fangpinsern committed:
Updated Developer guide and user guide

Updated user guide and developer guide with relevant diagrams and images.
Added new functionality to remove tags in the edit feature

FelixNWJ committed:
Update Developer Guide

fangpinsern committed:
Merge pull request #158 from iltep64/master

Various fixes for budget storage and bar display

fangpinsern committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/main

FelixNWJ committed:
Update FAQ, Glossary and create a List of Icons

FelixNWJ committed:
Introduce List of format and icons in Developer Guide

FelixNWJ committed:
Update PPP format list

MauriceTXS committed:
Updated notification feature and notification test case

FelixNWJ committed:
Solve merge conflict

MauriceTXS committed:
Merge branch 'master' into dev-guide

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

# Conflicts:
#	docs/DeveloperGuide.adoc

FelixNWJ committed:
Merge pull request #170 from MauriceTXS/dev-guide

Updated dev guide, notification feature and notification test case

FelixNWJ committed:
Fix checkstyle

MauriceTXS committed:
Merge pull request #168 from FelixNWJ/master

Developer Guide markup change

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

MauriceTXS committed:
Updated PPP

fangpinsern committed:
Edit command bug fixes

FelixNWJ committed:
Edit markup

FelixNWJ committed:
New suggest pane picture

FelixNWJ committed:
Merge pull request #171 from MauriceTXS/dev-guide

Updated PPP

MauriceTXS committed:
Updated Readme

MauriceTXS committed:
Merge pull request #172 from FelixNWJ/master

Update DG

MauriceTXS committed:
Merge branch 'master' into dev-guide

fangpinsern committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/main

FelixNWJ committed:
Update PPP

FelixNWJ committed:
Merge pull request #169 from fangpinsern/master

Updated Developer guide and user guide

MauriceTXS committed:
Merge pull request #174 from FelixNWJ/master

Update PPP

FelixNWJ committed:
Merge pull request #173 from MauriceTXS/dev-guide

Updated Readme

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

MauriceTXS committed:
Updated PPP

FelixNWJ committed:
Update AboutUs

FelixNWJ committed:
Merge pull request #175 from MauriceTXS/dev-guide

Updated PPP

iltep64 committed:
Final updates to UG, DG, PPP.

FelixNWJ committed:
Merge pull request #176 from iltep64/master

Final updates to UG, DG, PPP.

fangpinsern committed:
UserGuide Q&A update

fangpinsern committed:
UserGuide Q&A update

FelixNWJ committed:
Merge pull request #177 from fangpinsern/master

Updated UG

FelixNWJ committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W11-4/addressbook-level3

MauriceTXS committed:
Updated PPP

FelixNWJ committed:
Update DG

FelixNWJ committed:
Fix DG

FelixNWJ committed:
Merge pull request #178 from MauriceTXS/dev-guide

Updated PPP

MauriceTXS committed:
Merge pull request #179 from FelixNWJ/master

Update DG

fangpinsern committed:
Update Userguide

MauriceTXS committed:
Updated PPP and user guide

fangpinsern committed:
Update Userguide

MauriceTXS committed:
Updated user guide

fangpinsern committed:
Updated Userguide again

MauriceTXS committed:
Updated developer guide

iltep64 committed:
Fix typo in PPP and minor javadoc errors.

MauriceTXS committed:
Updated developer guide

FelixNWJ committed:
Update PPP

MauriceTXS committed:
Merge branch 'master' into dev-guide

FelixNWJ committed:
Merge pull request #180 from MauriceTXS/dev-guide

Updated PPP and user guide

fangpinsern committed:
Updated User Guide again

iltep64 committed:
Merge pull request #181 from FelixNWJ/master

Update PPP

FelixNWJ committed:
Merge pull request #182 from iltep64/master

Fix typo in PPP and minor javadoc errors.

FelixNWJ committed:
Merge pull request #183 from fangpinsern/master

User Guide update

MauriceTXS committed:
Merge branch 'master' into dev-guide

FelixNWJ committed:
Merge pull request #184 from MauriceTXS/dev-guide

Updated Developer Guide

fangpinsern committed:
Updated Developer Guide again

FelixNWJ committed:
Merge pull request #185 from fangpinsern/master

Updated Developer Guide again

FelixNWJ committed:
Slight Update in User Guide Report feature regarding calculation of Average Value

FelixNWJ committed:
Updated General Parts of the Developer Guide to fit new markup format style.

FelixNWJ committed:
Update Developer Guide Architectural Sections with reasons behind designs.

iltep64 committed:
Fix PPP naming issue.

MauriceTXS committed:
Merge pull request #187 from iltep64/master

Fix PPP naming issue.
